test.sh.in

```diff
@@ -262,42 +300,13 @@ setupVirtualEnv(){
 BASE_REPO_URL="${BASE_REPO_URL}"
 runTests() {
   set +e
+  netstat -ntlp
```

Comment:
do you want "echo $(netstat -ntlp)" instead ? otherwise, the output of netstat will not be shown up